packages/flutter/lib/src/cupertino/picker.dart

```diff
@@ -193,6 +190,11 @@ class CupertinoPicker extends StatefulWidget {
   /// A delegate that lazily instantiates children.
   final ListWheelChildDelegate childDelegate;
 
+  /// The [magnifier] widget overlaid on top of [ListWheelScrollView],
+  /// it will be vertically centered and will be the same size as the magnified item,
+  /// default to use [CupertinoPickerDefaultMagnifier].
```

Comment:
```
A widget overlaid on the picker to highlight the currently selected entry.

The [selectionOverlay] widget drawn above the [CupertinoPicker]'s picker wheel. 
It is vertically centered in the picker and is constrained to have the same height as the
center row. 

If unspecified, it defaults to a [CupertinoPickerDefaultMagnifier] which is a gray 
rounded rectangle overlay in the iOS 14 style. 
```

Word wrap as needed. In general, documentation need a 1 line summary (which is what appears when you hover over a field in IDEs' popup hints). 

I wouldn't mention ListWheelScrollView here since it's an implementation detail of this API and users probably won't know what it is.